CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 14.1.1 (10/23/23)
+
+* Fix the top bar breaking layout when the window is narrow in Connect [#33821](https://github.com/gravitational/teleport/pull/33821)
+* Limit Snowflake decompressed request to 10MB [#33764](https://github.com/gravitational/teleport/pull/33764)
```

Comment:
nit: we've always used past tense for changelog entries for whatever reason, so these two should be "Added" and "Limited"

Reply:
Good catch - updating now